pulpanalytics/models.py

```diff
@@ -27,7 +27,7 @@ class Meta:
         verbose_name_plural = "online content apps"
 
     def __str__(self):
-        return f"System={self.system.id}, Processes={self.processes}, Hosts={self.hosts}"
+        return f"System={self.system.system_id}, Processes={self.processes}, Hosts={self.hosts}"
```

Comment:
```suggestion
        return f"SystemID={self.system.system_id}, Processes={self.processes}, Hosts={self.hosts}"
```

I've made a real mess lol

Reply:
This is for console access only to be fair.

Reply:
It's for the django admin site.